colly_test.go

```diff
@@ -101,7 +103,6 @@ func newTestServer() *httptest.Server {
 
 	mux.Handle("/redirect", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		http.Redirect(w, r, "/redirected/", http.StatusSeeOther)
-
```

Comment:
I agree that all of these formatting changes are good, but let's keep only the relevant changes in the patch.

Reply:
Yep, sorry, I totally forgot about my auto-format settings when I did this pr. I'm using [gofumpt](https://github.com/mvdan/gofumpt) BTW.